internal/httputil/access_log.go

```diff
@@ -0,0 +1,91 @@
+package httputil
+
+import (
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/TykTechnologies/tyk-pump/analytics"
+	"github.com/TykTechnologies/tyk/request"
+)
+
+// AccessLogRecord is a representation of a transaction log in the Gateway
+type AccessLogRecord struct {
+	APIID    string
+	APIKey   string
+	OrgID    string
+	Latency  *analytics.Latency
+	Request  *http.Request
+	Response *http.Response
+}
+
+// NewAccessLogRecord returns an AccessLogRecord object
+func NewAccessLogRecord(apiID string, apiKey string, orgId string) *AccessLogRecord {
+	return &AccessLogRecord{
+		APIID:  apiID,
+		APIKey: apiKey,
+		OrgID:  orgId,
+	}
+}
```

Comment:
```suggestion
func NewAccessLogRecord(apiID string, apiKey string, orgId string) *AccessLogRecord {
	r := &AccessLogRecord{}
	r.Fields =  logrus.Fields{
		"APIID":  a.APIID,
		"APIKey": a.APIKey,
		"OrgID":  a.OrgID,
		"prefix": "access-log",
	}
	return r
}
```

I think the keys still need renaming (see other feedback)